src/definitions/modules/progress.js

```diff
@@ -260,16 +260,13 @@
 
                 get: {
                     text: function (templateText, index) {
-                        if (!index) {
```

Comment:
We have explicitely added this in the previous PRs...now you put in again? Can we please find a "final standard" :)